src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -37,9 +41,44 @@ public static JsonSerializerSettings GetSerializerSettings()
             return settings;
         }
 
+        /// <summary>
+        /// Customises the given serializer settings
+        /// using provider configuration.
+        /// Can be used by any provider, allowing the users to use a standard set of configuration attributes.
+        /// </summary>
+        /// <param name="set"></param>
+        /// <param name="config"></param>
+        /// <returns><see cref="JsonSerializerSettings" /></returns>
+        public static JsonSerializerSettings UpdateSerializerSettings(JsonSerializerSettings set, IProviderConfiguration config)
```

Comment:
I agree with the move, but why don't we put this and the `GetDefaultSerializerSettings` factory methods in some more generic class that it not related to IExternalSerializer, since it's being used for many purposes?